src/main/java/nl/tudelft/ti2206/group9/gui/Style.java

```diff
@@ -61,20 +61,23 @@
     /** PLAYER material used for the player. */
     public static final PhongMaterial PLAYER = new PhongMaterial();
 
-	/** Size of a button while hovering (relative to 1). */
+    /** Size of a button while hovering (relative to 1). */
     private static final double BUTTON_HOVER_SCALE = 1.2;
 
     /** Preferred width of buttons. */
     private static final int BUTTON_WIDTH = 120;
-    /** Popup font size. */
+    /** Popup globalFont size. */
     private static final int POPUP_TEXT = 11;
 
+    /** Font used to render everything. */
+    private static Font globalFont;
```

Comment:
I did at first, but PMD complained that I was using a redundant field initializer, meaning that the default value is null. And the font loads without problems, so I guess it works :)

<review,logic